tcp.c

```diff
@@ -473,3 +478,11 @@ int tcpdetach(tcpsock s) {
     mill_assert(0);
 }
 
+ipaddr tcpconnip(tcpsock s)
+{
```

Comment:
sorry, small style nitpick, this lib starts function's brace on same line as function declaration